WebRoot/css/dwt.css

```diff
@@ -2041,7 +2039,9 @@ INPUT.xform_field {
 .ZSelectAutoSizingContainer					{	height:2rem; width:auto;	}		/* TODO: SIZE ASSUMPTION! */
 
 .ZSelect									{	@Select@		@SelectSpacing@			}
-.ZSelect .ZWidgetTitle						{	padding-left:5px; text-align:left;		}
+.ZSelect .ZWidgetTitle						{	@SelectTitle@		}
```

Comment:
Minor: Indentation.

I think we should have one separate commit where we have standard editor based indentations for css files `dwt.css`, `common.css`, `_base4/skin.css/` and `zm.css` etc., so we don't end up in indentation issues anymore? I feel current indentation is not a standard and very confusing.

@silentsakky , @tarangkhandelwal , @SatyamKumarOnline, thoughts?

Thanks